website/src/_posts/2019-05-22-beam-2.13.0.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Apache Beam 2.13.0"
+date:   2019-05-22 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+        - goenka
+
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.13.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#2130-2019-05-21) for this release.<!--more-->
+For more information on changes in 2.13.0, check out the
+[detailed release notes](https://jira.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12345166).
+
+## Highlights
+
+### I/Os
+
+* Add support for a BigQuery custom sink for Python SDK.
+* Add support to specify a query in CassandraIO for Java SDK.
+* Add experimental support for cross-language transforms,
+  please see [BEAM-6730](https://issues.apache.org/jira/browse/BEAM-6730)
+* Add support in the Flink Runner for exactly-once Writes with KafkaIO
+
+### New Features / Improvements
+
+* Enable Bundle Finalization in Python SDK for portable runners.
+* Add support to the Java SDK harness to merge windows.
+* Add Kafka Sink EOS support on Flink runner.
+* Added a dead letter queue to Python streaming BigQuery sink.
+* Add Experimental Python 3.6 and 3.7 workloads enabled.
+  Beam 2.13 supports starting Dataflow pipelines under Python 3.6, 3.7, however 3.5 remains the only recommended minor version for Dataflow runner. In addition to announced 2.11 limitations, Beam typehint annotations are currently not supported on Python >= 3.6.
+
```

Comment:
Flink runner support for Flink 1.8 was added in this release.

Reply:
Added Flink 1.8 support to the post.